src/DynamicExpresso.Core/Parsing/Parser.cs

```diff
@@ -262,6 +262,12 @@ private Expression ParseComparison()
 				//			op.text, ref left, ref right, op.pos);
 				//}
 
+				if((IsNullableType(left.Type) || IsNullableType(right.Type)) && (GetNonNullableType(left.Type) == right.Type || GetNonNullableType(right.Type) == left.Type))
+                {
```

Comment:
Thanks! It looks great! Can you just fix a couple of indentation here and below? Project is configured with tabs...

Reply:
Done